app/routes/metrics.py

```diff
@@ -45,11 +45,11 @@ class AreasResponse(BaseModel):
 
 async def metric_id_param(
     metric_id: Annotated[
-        Literal["LossPersistence", "Coverage"],
-        fastapi.Path(description="metric you whish to query"),
+        str,
```

Comment:
What happens is that when performing tests to see if the collection is found or not, I identify that what this does is burn the name of the metric_id, meaning that if I place another metric in the endpoint, it will always indicate that it is not there because it is passing that burned parameter.

Reply:
that's te point. The backend will not process requests for metrics that don't exist

Reply:
you haven't addressed this point